be/src/service/mem_hook.cpp

```diff
@@ -241,9 +244,29 @@ std::atomic<int64_t> g_mem_usage(0);
 #define IS_BAD_ALLOC_CATCHED() false
 #endif
 
+#ifdef USE_JEMALLOC
+const size_t large_memory_alloc_report_threshold = 1073741824;
+inline thread_local bool skip_report = false;
+inline void report_large_memory_alloc(size_t size) {
+    if (size > large_memory_alloc_report_threshold && !skip_report) {
+        skip_report = true; // to avoid recursive output log
```

Comment:
It's unnecessary to skip this log

Reply:
> It's unnecessary to skip this log

memory may be allocated when outputting the log. if a large memory allocation is encountered, it will recurse infinitely…This is to avoid this situation(although it is unlikely to happen)

Reply:
oh I see